src/components/BoraPortal.tsx

```diff
@@ -74,6 +78,94 @@ function BoraPortal() {
     }
   }
 
+  const getAccountInfoCallback = useCallback(async () => {
+    if (!face) {
+      return null;
+    }
+
+    const { address, balance, user } = await getAccountInfo(face, network!);
+
+    console.group('[Account Information]');
+    console.log('Balance:', balance);
+    console.log('Address:', address);
+    console.log('Current user:', user);
+    console.groupEnd();
+
+    setAccount({ address, balance, user });
+  }, [face, network, setAccount]);
+
+  async function boraLogin() {
+    if (!bappUsn) {
+      console.error('bappUsn을 입력해주세요.');
+      return;
+    }
+    try {
+      const signatureMessage = `boraconnect:${bappUsn}`;
+      const res = await face?.auth.boraLogin({
+        bappUsn,
+        signature: createSignature(signatureMessage, prvKey),
+      });
+      // setSession((res as any).session);
+      setIsLoggedIn(true);
+      getAccountInfoCallback();
+      console.log('Login response:', res);
+    } catch (e) {
+      if (e.isFaceError && e.code === 4001) {
+        console.log('User rejected!');
+        return;
+      }
+      throw e;
+    }
+  }
+
+  async function boraDirectSocialLogin(provider: LoginProviderType) {
+    try {
+      if (!bappUsn) {
+        console.error('bappUsn을 입력해주세요.');
+        return;
+      }
+      const signatureMessage = `boraconnect:${bappUsn}`;
+      const res = await face?.auth.boraDirectSocialLogin(
+        {
+          bappUsn,
+          signature: createSignature(signatureMessage, prvKey),
+        },
+        provider
+      );
+      console.log('Social Login response:', res);
+      setIsLoggedIn(true);
+      getAccountInfoCallback();
+    } catch (e) {
+      if (e.isFaceError && e.code === 4001) {
+        console.log('User rejected!');
+        return;
+      }
+      throw e;
+    }
+  }
+
+  async function boraLoginWithIdToken(provider: LoginProviderType) {
+    if (!bappUsn) {
+      console.error('bappUsn을 입력해주세요.');
```

Comment:
Since this repository is public, our customer can run this code.
Let's use English.

Reply:
I'will fix it. thanks.